modules/onlyfans.py

```diff
@@ -865,7 +865,14 @@ def compare_metadata(new_metadata: create_metadata, old_metadata: create_metadat
     return new_metadata
 
 # Scrapes the API for content
-
+unknowncount = 0
+def getValueIfKey(key, dictv):
+    global unknowncount
+    if key in dictv:
+        return dictv[key]
+    else:
+        unknowncount+=1
+        return 'unknown{unkowncount}'
```

Comment:
typo? shouldn't it be `{unknowncount}` not `{unkowncount}`?

Reply:
You are right. I have fixed that. I did this PR in a rush.